tracer/build/crank/loader.conf

```diff
@@ -0,0 +1,6 @@
+#Profiler
+PROFILER;{BD1A650D-AC5D-4896-B64F-D6FA25D6B26A};linux-x64;../profiler-home-linux/Datadog.AutoInstrumentation.Profiler.Native.x64.so
+
+#Tracer
+TRACER;{50DA5EED-F1ED-B00B-1055-5AFE55A1ADE5};linux-x64;../tracer-home-linux/Datadog.Trace.ClrProfiler.Native.so
+
```

Comment:
Should we add the other values from `tracer/src/Datadog.AutoInstrumentation.NativeLoader/loader.conf` too?

Reply:
I didn't because it might give people the idea that it was possible to use this with a Windows instances, when that's untested.